### ISSUE TITLE
docs: APM "Personal data" built-in data filter doc needs an  update

### ISSUE THREAD

**Issue by mlliarm** [elastic/apm-server#13622]
Hello,

[In the documentation it writes](https://www.elastic.co/guide/en/observability/8.14/apm-filtering.html#apm-filters-personal-data):

> The capturing of this data can be turned off by setting Capture personal data to false.

Here's the flag mentioned in the quote, within `Integrations > Elastic APM > Elastic APM`:

![image](https://github.com/elastic/apm-server/assets/5833270/8c4cf4d6-d4a9-48c2-91c5-e5601b9d5a8f)

But the quote isn't accurate, as the capturing of client data such as `client.ip` cannot be deactivated and as a result such data appear in the logs.

As of now the only way to block such information entering the logs is reducting it using [a filter on the client side](https://www.elastic.co/guide/en/apm/agent/nodejs/current/agent-api.html#apm-add-filter).

So, the above line can be confusing as a user might expect that by switching the "Capture personal data" to false no personal data such as `client.ip` will go through the APM server into Kibana.

Summing up, I believe there should be a more explicit explanation of what is not being captured when this flag is deactivated.

Thanks

**Comment by raultorrecilla** [comment]
moving it from it-105 to it-108
